lib/ui/widgets/link_web_view.dart

```diff
@@ -0,0 +1,44 @@
+// FLUTTER / DART / THIRD-PARTIES
+import 'package:flutter/material.dart';
+
+// MODELS
+import 'package:notredame/core/models/quick_link.dart';
+
+// OTHER
+import 'package:webview_flutter/webview_flutter.dart';
+
+class LinkWebView extends StatefulWidget {
+  final QuickLink _links;
+
+  const LinkWebView(this._links);
+
+  @override
+  _LinkWebViewState createState() => _LinkWebViewState();
+}
+
+class _LinkWebViewState extends State<LinkWebView> {
+  bool isLoading = true;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(widget._links.name),
+      ),
+      body: Stack(
+        children: <Widget>[
+          WebView(
+            initialUrl: widget._links.link,
+            javascriptMode: JavascriptMode.unrestricted,
+            onPageFinished: (finish) {
+              setState(() {
+                isLoading = false;
+              });
```

Comment:
is there a onPageError or something like that? so maybe if the quicklinks doesn't load we could launch a toast to warn the user and log an error using analyticsService.

Reply:
Nothing in the WebView seems to handle that. I found this property :  onWebResourceError, but it is not called when the link is not valid.